recognition/include/pcl/recognition/quantized_map.h

```diff
@@ -121,20 +122,24 @@ namespace pcl
         }
       }
 
-      void 
+      void
       deserialize (std::istream & stream)
       {
         int width;
         int height;
 
         stream.read (reinterpret_cast<char*> (&width), sizeof (width));
-        stream.read (reinterpret_cast<char*> (&height), sizeof (height));
-
+        PCL_CHECK_IO_STREAM(stream, "width");
         width_ = static_cast<std::size_t> (width);
+
+        stream.read (reinterpret_cast<char*> (&height), sizeof (height));
+        PCL_CHECK_IO_STREAM(stream, "height");
         height_ = static_cast<std::size_t> (height);
 
         int num_of_elements;
         stream.read (reinterpret_cast<char*> (&num_of_elements), sizeof (num_of_elements));
+        PCL_CHECK_IO_STREAM(stream, "num_of_elements");
```

Comment:
Your checking seems inconsistent. You're checking for this condition in `recognition/src/dotmod.cpp` but not here (and multiple other places)

Reply:
Yeah my bad, will make it uniform by adding this check at the places it isn't there currently